impl_core/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -180,6 +182,18 @@ ViewData toViewData(Timestamp now, State state) {
       }
     }
 
+    /**
+     * This method attemps to migrate this view into a reasonable state in the event of time going
+     * backwards.
+     */
+    private void handleTimeRewinds(Timestamp now) {
+      if (now.compareTo(start) < 0) {
```

Comment:
Should this compare to the last recorded timestamp from L156 instead of `start`?

Reply:
This class isn't recording timestamps.  See L146 where it just sets each data point its collected to `now`.  While it *looks* like the `MutableAggregation` is receiving timestamps, it's actually dropping them and just storing `Sum` or `Count` in most cases.